contracts/Orchestrator.sol

```diff
@@ -3,6 +3,7 @@ pragma solidity 0.7.6;
 import "./_external/Ownable.sol";
 
 import "./UFragmentsPolicy.sol";
+import { ECDSA } from "./_external/ECDSA.sol";
```

Comment:
You could directly import from `openzeppelin/contracts/utils/cryptography/ECDSA.sol`, rather than copy the code into the repo.

Reply:
**Some background:**
We copied dependency code in for `UFragments` and `UFragmentsPolicy` to preserve the storage layout (since the first version was built on solidity 0.4.24, openzeppelin v2.2 and later versions of openzeppelin altered the storage layout).I dont see us doing this ever again. 

We can add external libraries as a dependency using `yarn` and just import required contracts directly.